src/DynamoPackages/PackageDirectoryBuilder.cs

```diff
@@ -127,8 +127,7 @@ private void WritePackageHeader(Package package, IDirectoryInfo rootDir)
             var pkgHeader = PackageUploadBuilder.NewRequestBody(package);
 
             // build the package header json, which will be stored with the pkg
-            var jsSer = new JsonSerializer();
-            var pkgHeaderStr = jsSer.Serialize(pkgHeader);
+            var pkgHeaderStr = JsonSerializer.Serialize(pkgHeader);
```

Comment:
switched to system.text.json for package related serialization as that is what Greg.dll now uses.